src/Badge/Badge.js

```diff
@@ -39,8 +78,16 @@ Status.propTypes = {
 };
 
 export const Counter = ({ notification, children, className, ...props }) => {
+    const counterClasses = classnames(
+        'fd-counter',
+        {
+            'fd-counter--notification': notification
+        },
+        className
+    );
+
     return (
-        <span aria-label='Unread count' className={`fd-counter${notification ? ' fd-counter--notification' : ''}${className ? ' ' + className : ''}`}
+        <span aria-label='Unread count' className={counterClasses}
```

Comment:
The `props` spread should come first or `aria-label` could be overridden by the consumer.